felt/src/arbitrary.rs

```diff
@@ -1,52 +1,41 @@
-use num_bigint::BigUint;
+use lambdaworks_math::{field::element::FieldElement, unsigned_integer::element::UnsignedInteger};
 use num_traits::Zero;
 use proptest::prelude::*;
 
-use crate::{
-    bigint_felt::{FeltBigInt, FIELD_HIGH, FIELD_LOW},
-    Felt252,
-};
+use crate::{Felt252, FIELD_HIGH, FIELD_LOW};
 
-fn any_felt_big_int<const PH: u128, const PL: u128>() -> impl Strategy<Value = FeltBigInt<PH, PL>> {
-    (0..=PH)
+/// Returns a [`Strategy`] that generates any valid Felt252
+fn any_felt252() -> impl Strategy<Value = Felt252> {
```

Comment:
At this point we could think about moving this to lambdaworks.

Reply:
Yes, thanks for the suggestion! Created an issue: https://github.com/lambdaclass/lambdaworks/issues/414